x-pack/plugins/security_solution/public/management/pages/policy/view/policy_details.test.tsx

```diff
@@ -123,35 +124,29 @@ describe('Policy Details', () => {
 
     it('should display back to list button and policy title', () => {
       policyView.update();
-      const pageHeaderLeft = policyView.find(
-        'EuiPageHeaderSection[data-test-subj="pageViewHeaderLeft"]'
-      );
 
-      const backToListButton = pageHeaderLeft.find('EuiButtonEmpty');
-      expect(backToListButton.prop('iconType')).toBe('arrowLeft');
-      expect(backToListButton.prop('href')).toBe(endpointListPath);
-      expect(backToListButton.text()).toBe('Back to endpoint hosts');
+      const backToListLink = policyView.find('LinkIcon[dataTestSubj="policyDetailsBackLink"]');
+      expect(backToListLink.prop('iconType')).toBe('arrowLeft');
+      expect(backToListLink.prop('href')).toBe(endpointListPath);
+      expect(backToListLink.text()).toBe('Back to endpoint hosts');
 
-      const pageTitle = pageHeaderLeft.find('[data-test-subj="pageViewHeaderLeftTitle"]');
```

Comment:
I think you need to make a similar change to the Endpoint list test to fix the other test failures

`x-pack/test/security_solution_endpoint/apps/endpoint/endpoint_list.ts`

Reply:
Same as response above.